python/jupytergis_core/src/jgisplugin/plugins.ts

```diff
@@ -181,11 +184,65 @@ const activate = (
       category: 'JupyterGIS'
     });
 
+    // Layers only
+    palette.addItem({
+      command: CommandIDs.newRasterLayer,
+      category: 'JupyterGIS'
+    });
+
     palette.addItem({
       command: CommandIDs.newVectorLayer,
       category: 'JupyterGIS'
     });
 
+    palette.addItem({
+      command: CommandIDs.newHillshadeLayer,
+      category: 'JupyterGIS'
+    });
+
+    // Layer and group actions
+    palette.addItem({
+      command: CommandIDs.renameLayer,
+      category: 'JupyterGIS'
+    });
+
+    palette.addItem({
+      command: CommandIDs.removeLayer,
+      category: 'JupyterGIS'
+    });
+
+    palette.addItem({
+      command: CommandIDs.renameGroup,
+      category: 'JupyterGIS'
+    });
+
+    palette.addItem({
+      command: CommandIDs.removeGroup,
+      category: 'JupyterGIS'
+    });
+
+    palette.addItem({
+      command: CommandIDs.moveLayerToNewGroup,
+      category: 'JupyterGIS'
+    });
+
+    // Source actions
+    palette.addItem({
+      command: CommandIDs.renameSource,
+      category: 'JupyterGIS'
+    });
+
+    palette.addItem({
+      command: CommandIDs.removeSource,
+      category: 'JupyterGIS'
+    });
```

Comment:
Wondering if we want to add these ones to the palette.
These commands need to have an item selected in side panel. But when the item is selected, it seems easier to type `Del` or `F2` instead of opening the command palette and search for the command...